File: iTSoft.CRM.UI/src/app/_models/clientViewModel.ts
import { ClientMaster } from "./clientMaster";
import { ContactPersonMaster } from "./contactPerson";
import { OrganizationMaster } from "./organization";

export class ClientViewModel {
    ClientMaster: ClientMaster;
    OrganizationMaster: OrganizationMaster;
    ContactPersonMaster: ContactPersonMaster;
    ContactPersonMasters: ContactPersonMaster[];

    constructor()
    {
        this.ClientMaster = new ClientMaster();
        this.OrganizationMaster = new OrganizationMaster();
        this.ContactPersonMaster = new ContactPersonMaster();
        this.ContactPersonMasters = [];
    }
}

File: iTSoft.CRM.Web_Old/iTSoft.CRM.Data/ViewModel/ClientViewModel.cs
using iTSoft.CRM.Data.Entity.Master;
using iTSoft.CRM.Data.Entity.Process;
using System;
using System.Collections.Generic;
using System.Text;

namespace iTSoft.CRM.Data.ViewModel
{
    public class ClientViewModel
    {
        public ClientMaster ClientMaster { get; set; }
        public OrganizationMaster OrganizationMaster { get; set; }
        public List<ContactPersonMaster> ContactPersonMasters { get; set; }
    }
}
